client/task-list/task-list.js

```diff
@@ -246,6 +247,26 @@ export const TaskList = ( {
 		return <div className="woocommerce-task-dashboard__container"></div>;
 	}
 
+	const expandLabel = sprintf(
+		/* translators: %i = number of hidden tasks */
+		__( 'Show %i more tasks.', 'woocommerce-admin' ),
+		listTasks.length - 2
+	);
+	const collapseLabel = __( 'Show less', 'woocommerce-admin' );
+	const ListComp = name === 'task_list' ? List : CollapsibleList;
```

Comment:
It's probably not be worth doing now, but a refactor to avoid this level of knowledge in the component would be nice.

Reply:
Yeah, I agree, although currently this component has quite a bit of logic between `task_list` and `extended_tasks`. I suppose we could write another abstraction component with the differences as prop options.